dask_jobqueue/tests/test_slurm.py

```diff
@@ -48,6 +48,7 @@ def test_job_script():
     with SLURMCluster(walltime='00:02:00', processes=4, cores=8, memory='28GB') as cluster:
 
         job_script = cluster.job_script()
+        assert job_script.startswith("#!/bin/bash")
```

Comment:
This should also go into the tests for the other schedulers.

Reply:
I believe this should only go in https://github.com/dask/dask-jobqueue/blob/master/dask_jobqueue/tests/test_jobqueue_core.py as this is not specific to any scheduler.